icu4c/source/common/utf_impl.cpp

```diff
@@ -55,7 +55,7 @@
  * -finish:
  * (BSR: Bit Scan Reverse, scans for a 1-bit, starting from the MSB)
  */
-extern "C" U_EXPORT const uint8_t
+U_CAPI const uint8_t
```

Comment:
Kind of curious about this one-- if this is just in the .cpp file, what's U_CAPI doing for you?  Is somebody else actually using this despite its not being in a header?

Reply:
I think `utf8_countTrailBytes` is used in tests. `U_CAPI` expands to `U_CFUNC U_EXPORT` which expands to `extern "C" U_EXPORT` so its the same annotation. I updated this to match with [the updated header declaration in this PR](https://github.com/unicode-org/icu/pull/2118/files#diff-5cc093838cf40a8ed1ba9f84d4c32e22789d1c77dd86209d366c02f683024a3fR296).

Reply:
I see.  Thanks for the explanation!